resource_lists/the survivalist/resources.yaml

```diff
@@ -0,0 +1,2004 @@
+authors:
+  jojoblackFr: https://github.com/jojoblackFr
+game_version: 1.1.13.565.740066
+index_page_display_name: The Survivalist
```

Comment:
Is the game name `The Survivalist` or `The Survivalists`.
The logo and steam page says `The Survivalists` with an `s`
If it does have an s then we should probably update the folder name to match